src/crypto/tests/CHIPCryptoPALTest.cpp

```diff
@@ -118,6 +124,12 @@ using TestHMAC_sha = HMAC_shaHSM;
 using TestHMAC_sha                      = HMAC_sha;
 #endif
 
+#if CHIP_CRYPTO_PSA
+using TestSessionKeystore = PSASessionKeystore;
+#else
+using TestSessionKeystore               = RawKeySessionKeystore;
```

Comment:
Spacing here is very odd

Reply:
Yeah, I don't know why clang-format formats it that way...